vulture_os/toolkit/api_parser/reachfive/reachfive.py

```diff
@@ -174,4 +197,14 @@ def execute(self):
                 # Replace "Z" by "+00:00" for datetime parsing
                 # No need to make_aware, date already contains timezone
                 self.frontend.last_api_call = datetime.fromisoformat(last_datetime.replace("Z", "+00:00"))
+
+            if cpt % 10000 == 0:
+                # Sleep 10 sec every 10 000 fetched logs to avoid too many requests
+                time.sleep(10)
```

Comment:
```suggestion
                self.evt_stop.wait(10.0)
```
prefer using a .wait on the evt_stop event, allowing to let the collector stop early if asked to (instead of always waiting 10 seconds, even when asked to stop prematurely)